dspace-server-webapp/src/main/java/org/dspace/app/rest/BundleUploadBitstreamController.java

```diff
@@ -100,6 +100,9 @@
         if (bundle == null) {
             throw new ResourceNotFoundException("The given uuid did not resolve to a Bundle on the server: " + uuid);
         }
+        if (uploadfile == null) {
+            throw new UnprocessableEntityException("No file was provided");
```

Comment:
it should return 400 according to the discussion in https://jira.lyrasis.org/browse/DS-4428

Reply:
This is based on https://github.com/DSpace/Rest7Contract/pull/71/files
Does the change of desired functionality in https://jira.lyrasis.org/browse/DS-4428 imply this PR is no longer needed?